### ISSUE TITLE
[Update Request]: Update TortoiseSVN.TortoiseSVN to 1.14.6

### ISSUE THREAD

**Issue by MagicAndre1981** [microsoft/winget-pkgs#132129]
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

TortoiseSVN.TortoiseSVN

### Package Version

1.14.6

### Please describe the changes you would like to see

**Version 1.14.6**

- BUG: context menu for a directory background in
       Win11 when not using the first tab in explorer
       used the wrong path. (Stefan)
- BUG: the revision graph did not show the HEAD revision
       if not all revisions were shown. (Stefan)
- BUG: context menu items in Win11 did show up but
       didn't invoke the command. (Stefan)
- BUG: the repository tree control in the Checkout "Choose
       items" dialog didn't resize properly when resizing
       the window. (Daniel Sahlberg)
- BUG: TMerge didn't properly restore line numbers when undoing
       pasting multiple lines. (Daniel Sahlberg)
- BUG: Project monitor duplicated the list of Users to ignore
       every time a project was edited. (Daniel Sahlberg)
- BUG: Project monitor kept an error status on a project that had
       temporary connection issues, even after the next connection
       was successful. (Daniel Sahlberg)
- NEW: added a menu item in Project monitor to check one specific
       project. (Daniel Sahlberg)
- NEW: added an advanced setting to control the default value
       of Allow mixed revisions (not recommended) in the Merge
       options dialog. (Daniel Sahlberg)
- BUG: TortoiseBlame didn't restore saved window position. (Stefan)
- BUG: Repository-Urls with ipv6 addresses were not properly
       handled. (Stefan)

**Download:**
**x86:** 
https://sourceforge.net/projects/tortoisesvn/files/1.14.6/Application/TortoiseSVN-1.14.6.29673-win32-svn-1.14.3.msi/download

**x64**
https://sourceforge.net/projects/tortoisesvn/files/1.14.6/Application/TortoiseSVN-1.14.6.29673-x64-svn-1.14.3.msi/download

**ARM64**
https://sourceforge.net/projects/tortoisesvn/files/1.14.6/Application/TortoiseSVN-1.14.6.29673-ARM64-svn-1.14.3.msi/download

**Comment by mdanish-kh** [comment]
Close with reason: Resolved in PR #132147;

**Comment by logarrhythmic** [comment]
This version doesn't get installed when I run `winget update TortoiseSVN.TortoiseSVN`. `winget search tortoisesvn` returns the package for version 1.14.29465 a.k.a. 1.14.5.29465 if going by the sourceforge installer filename. `winget search tortoisesvn --versions` shows that 1.14.6.29673 is present, but presumably considered older because 6 < 29465. The re-addition of the minor version number seems to be making winget think this isn't the newest version, so it doesn't get installed by the update command.
```
PS C:\> winget search tortoisesvn
Name        Id                      Version
----------------------------------------------
TortoiseSVN TortoiseSVN.TortoiseSVN 1.14.29465

PS C:\> winget search tortoisesvn --versions
Found TortoiseSVN [TortoiseSVN.TortoiseSVN]
Version
------------
1.14.29465
1.14.29387
1.14.6.29673
1.10.0.28176
```
In case another user ends up here through a web search before this is fixed, `winget install TortoiseSVN.TortoiseSVN --source winget --version 1.14.6.29673 --force` should work - the TortoiseSVN installer seems to handle removing the older version even if winget doesn't think this is an update.
